src/main/java/com/squareup/javapoet/Util.java

```diff
@@ -109,4 +110,19 @@ public static void requireExactlyOneOf(Set<Modifier> modifiers, Modifier... mutu
   public static boolean hasDefaultModifier(Collection<Modifier> modifiers) {
     return DEFAULT != null && modifiers.contains(DEFAULT);
   }
+
+  public static String identifier(String part) {
```

Comment:
Alright. Will add tests that show the intended (and invalid) input strings. Your `$T...` comment does not apply, as `$T` is already resolved and every varargs method emitted didn't throw an exception for the entire test suite. You can't statically import a sole type name, can you?